src/main/kotlin/server/Yegg.kt

```diff
@@ -28,6 +28,8 @@ object Yegg {
     private const val DISCONNECT_MSG = "** Disconnected **"
     const val HUH_MSG = "I don't understand that."
 
+    private var inTestMode = false
```

Comment:
It's a bummer that you need to do this.  Could it be avoided with `mockkObject(..)` in your tests?